backend/infrahub/git/integrator.py

```diff
@@ -214,7 +214,7 @@ async def import_jinja2_transforms(
         local_transforms: dict[str, InfrahubRepositoryJinja2] = {}
 
         # Process the list of local Jinja2 Transforms to organize them by name
-        await self.log.info(f"Found {len(config_file.jinja2_transforms)} Jinja2 transforms in the repository")
+        log.info(f"Found {len(config_file.jinja2_transforms)} Jinja2 transforms in the repository")
```

Comment:
FYI, if we are using the standard logger, the log will not show up in prefect
Considering the current size of this PR I think we should revisit this part later but at some point we'll need to use Prefect logger

Reply:
Right, I think we would need a `get_log` utility function that returns either server logger or prefect logger depending on the context.